clerk/sessions_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"testing"
 )
 
-func TestSessionsService_ListAll_happyPath(t *testing.T) {
+func TestSessionsService_ListAll_happyPath_noParams(t *testing.T) {
```

Comment:
🔧 Could you please add 2 tests:
- `TestSessionsService_ListAll_pagination_and_filtering_params`
- `TestSessionsService_ListAll_pagination_and_filtering_empty_params`

test to assert that the added functionality works as expected?